weave/guides/tracking/otel.mdx

```diff
@@ -11,6 +11,15 @@ Weave supports ingestion of OpenTelemetry compatible trace data through a dedica
 **Path**: `/otel/v1/traces`
 **Method**: POST
 **Content-Type**: `application/x-protobuf`
+**Base URL**: The base URL for the OTEL trace endpoint depends on your W&B deployment type:
+
+- Multi-tenant Cloud:  
+  `https://traces.wandb.ai/otel/v1/traces`
+
+- Dedicated Cloud and self-mamaged instances:
```

Comment:
Self-Managed (that casing) uses whatever the deployment's URL is, totally at the customer's whim. See https://docs.wandb.ai/platform/hosting/self-managed/bare-metal#verify-your-installation for one way to handle placeholders.